src/frontend/src/optimizer/plan_node/logical_agg.rs

```diff
@@ -33,9 +33,32 @@ use crate::catalog::table_catalog::TableCatalog;
 use crate::expr::{AggCall, Expr, ExprImpl, ExprRewriter, ExprType, FunctionCall, InputRef};
 use crate::optimizer::plan_node::utils::TableCatalogBuilder;
 use crate::optimizer::plan_node::{gen_filter_and_pushdown, LogicalProject};
-use crate::optimizer::property::{Order, RequiredDist};
+use crate::optimizer::property::{Direction, Order, RequiredDist};
 use crate::utils::{ColIndexMapping, Condition, Substitute};
 
+/// See also [`crate::expr::AggOrderByExpr`]
+#[derive(Clone)]
+pub struct PlanAggOrderByField {
+    pub input: InputRef,
+    pub direction: Direction,
+    pub nulls_first: Option<bool>,
+}
```

Comment:
Technically, `nulls_first: bool` should also be part of `Direction`, and this would become the same as `FieldOrder`.

As that might be a big refactor, we can leave a TODO here to replace `PlanAggOrderByField` with enhanced `FieldOrder`.